documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -11,46 +11,329 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
-** _SubmodelElementList_ is serialized as a named JSON array with $\{SubmodelElementList/idShort} as the name of the containing JSON property. The elements in the JSON array are the ValueOnly serializations of the elements contained in the SubmodelElementList while preserving the order, i.e. index n in the JSON array is the ValueOnly serialization of the element at index n of the SubmodelElementList.
-** _ReferenceElement_ is serialized as $\{ReferenceElement/idShort}: $\{ReferenceElement/value} where $\{ReferenceElement/value} is the serialization of the _Reference_ class.
-** _RelationshipElement_ is serialized as named JSON object with $\{RelationshipElement/idShort} as the name of the containing JSON property. The JSON object contains two JSON properties. The first is named “first”. The second is named “second”. Their corresponding values are $\{RelationshipElement/first} resp. $\{Relationship/second}. The values are serialized according to the serialization of a _ReferenceElement_ (see above).
-** _AnnotatedRelationshipElement_ is serialized according to the serialization of a _ReleationshipElement_ (see above). Additionally, a third named JSON object is introduced with “annotations” as the name of the containing JSON property. The value is $\{AnnotatedRelationshipElement/annotations}. The values of the array items are serialized depending on the type of the annotation data element.
-** _Entity_ is serialized as named JSON object with $\{Entity/idShort} as the name of the containing JSON property. The JSON object contains three JSON properties. The first is named “statements” $\{Entity/statements} and contains an array of the serialized submodel elements according to their respective serialization mentioned in this clause. The second is named either “globalAssetId” or “specificAssetId” and contains either a _Reference_ (see above) or a _SpecificAssetId_. The third property is named “entityType” and contains a string representation of $\{Entity/entityType}.
-** _BasicEventElement_ is serialized as named JSON object with $\{BasicEventElement/idShort} as the name of the containing JSON property. The JSON object contains one JSON property named “observed” with the corresponding value of $\{BasicEventElement/observed} as the standard serialization of the _Reference_ class.
-** _SpecificAssetId_ is serialized as named JSON object with three JSON properties named as the attributes of _SpecificAssetId._
+[[table-value-data-attributes]]
+.Value-Only Attributes
+[%autowidth, width="100%", cols="48%,52%",options="header",]
+|===
+|*Class Name* |*Fields for Value-Only representationfootnote:[ attribute in parenthesis (like e.g. (value)) means that the attribute name is not displayed, only its Value-Only serialization. 
+For the other attributes the attribute itself will be part of the serialization]*
+2+|*Identifiables*
+|AssetAdministrationShell a|assetInformation, submodels
```

Comment:
removed attributes and added note that there is no serialization. If needed we can introduce it later.

Why assetInformation used in valueOnly : because the rest is Metainformation. We cannot change it because of backward compatibility. I added footnote for it.

Reply:
It seems it did mix up things and you are correct, assetInformation is **not** part of metadata serialization and therefore there is no contradiction in having it as part of valueOnly serialization.

However, I still recommend removing this section completely if it is not used because it is just cluttering the document without bringing any benefits. If kept, I propose to move assetInformation to the metadata serialization format instead of valueOnly as I would argue this data is rather considered metadata than a value.

In the end, if you really insist on keeping things as-are, I could also live with that as it has no actual effect because it is never used.

Reply:
I would not agree that assetInformation is metadata in the senses of $metadata: alle data in AAS is metadata in some sense but some are metadata for the asset being represented (assetInformation) and some for the AAS as object created by a data provider etc. (like e.g. versioning).

Reply:
I get your point. However, if the idea is to divide all properties into either metadata or value, I would consider `assetInformation` to rather be metadata than value.